Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/dao/impl/TrustBundleDaoImpl.java

```diff
@@ -158,8 +157,11 @@ public TrustBundle getTrustBundleByName(String bundleName) throws ConfigurationS
                 }
             }
         } catch (NoResultException e) {
+            LOG.warn("No results found: " + e.getLocalizedMessage());
+            LOG.trace("No results found: " + e.getLocalizedMessage(), e);
```

Comment:
Why all the hate on "warn" level today, lol?  This code maybe should be handled differently, but I don't believe this case is an error.